pkg/cmd/secret/set/set.go

```diff
@@ -187,6 +199,7 @@ func setRun(opts *SetOptions) error {
 		if err != nil {
 			return err
 		}
+
```

Comment:
TODO: Probably move the base repo resolution above the secret prompts so that it happens before the user goes to any trouble:

This is not great:

```
➜ ~/workspace/cli/bin/gh secret set foo
? Paste your secret: ***

? Select a base repo  [Use arrows to move, type to filter]
  williammartin-test-org/test-repo
> williammartin/test-repo
```